avatar+badge/avatar.css

```diff
@@ -0,0 +1,155 @@
+@import "../for-import/reset.css";
+@import "../for-import/colors.css";
+
+/* round avatar */
+.avatar-round {
+    border-radius: 50%;
+}
+
+/* square avatar */
+.avatar-square-lg {
+    border-radius: 0.9rem;
+}
+
+/* square avatar medium size */
+.avatar-square-md {
+    border-radius: 10px;
+}
+
+/* square avatar small size */
+.avatar-square-sm {
+    border-radius: 5px;
+}
+
+/*size of avatar */
+.avatar-large {
+    width: 7.1rem; /*115px */
```

Comment:
hey ruchi , in css add  html { "font-size:62.5%" } , because this would set html font-size to 10px. your calculation would become easier because "62.5% * 16px = 10px" . so here for 115px you could rewrite as 11.5 rem where 1 rem is 10px

Reply:
Thanks Mohammed-sas, I just checked with Atul and it would be okay to implement this at the end. All other changes are done :)